feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.html

```diff
@@ -19,6 +19,13 @@
     >{{ getLabel(expMode, attribute.label, attribute.name) }}</span
   ></label
 >
+<cx-configurator-show-more
```

Comment:
Isn't this breaking?

Reply:
Only for CPQ where we don't know of any customer. For CPS it's guided by a feature flag